fuel-coroutines/src/main/kotlin/com/github/kittinunf/fuel/coroutines/Coroutines.kt

```diff
@@ -23,22 +21,43 @@ private suspend fun <T : Any, U : Deserializable<T>> Request.await(
             }
         }
 
-
 suspend fun Request.awaitByteArrayResponse(
         scope: CoroutineContext = CommonPool
-): Triple<Request, Response, Result<ByteArray, FuelError>> =
-        await(byteArrayDeserializer(), scope)
+): Triple<Request, Response, Result<ByteArray, FuelError>> = await(byteArrayDeserializer(), scope)
+
+suspend fun Request.asyncByteArrayResponse(
```

Comment:
is there a  reason for changing the name of this function?

Reply:
I didn't change it.

Reply:
sorry I see that